pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -3,22 +3,22 @@
     "Microsoft.CodeAnalysis.CSharp": "1.3.0",
     "Microsoft.CodeAnalysis.VisualBasic": "1.3.0",
```

Comment:
I'm fine only updating some of these but I do think we should probably update all these at some point.